### PR TITLE
testfast

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
release fast test runtime and node. 

can normal node run testfast runtime ? YES. So need to use testfast to export chainspec. and run it with release node.

can testfast node run release runtimes? YES.

can testfast runtime be upgraded to release and back? YES

what is testfast = test is that Origin are relaxed , fast means configs for less time to wait (future will do 4 second block time instead of 12)

### CODE REVIEW

**Review by obsessed-cake — DISMISSED** [review]

File: code/parachain/runtime/picasso/src/ibc.rs [hunk 1]
```diff
@@ -134,18 +134,32 @@ impl pallet_ibc::Config for Runtime {
...
+	type AdminOrigin = EnsureRootOrOneThirdNativeTechnical;
+	#[cfg(feature = "testnet")]
+	type FreezeOrigin = EnsureRootOrOneThirdNativeTechnical;
+	#[cfg(feature = "testnet")]
+	type TransferOrigin = system::EnsureSigned<Self::IbcAccountId>;
+	#[cfg(feature = "testnet")]
+	type RelayerOrigin = system::EnsureSigned<Self::IbcAccountId>;
+
+	#[cfg(not(feature = "testnet"))]
+	type AdminOrigin = EnsureRootOrOneThirdNativeTechnical;
```
Comment (andresmechali) [inline code/parachain/runtime/picasso/src/ibc.rs:155]: This should be 2/3 no?

File: code/parachain/runtime/picasso/src/ibc.rs [hunk 2]
```diff
@@ -134,18 +134,32 @@ impl pallet_ibc::Config for Runtime {
...
+	#[cfg(not(feature = "testnet"))]
+	type FreezeOrigin = EnsureRootOrOneThirdNativeTechnical;
```
Comment (andresmechali) [inline code/parachain/runtime/picasso/src/ibc.rs:157]: Also this

**Review by andresmechali — APPROVED** [review]

File: code/parachain/runtime/picasso/src/ibc.rs (hunk 1 above)
Comment (kkast) [reply code/parachain/runtime/picasso/src/ibc.rs:155]: agreed

**Review by kkast — CHANGES_REQUESTED** [review]

File: code/parachain/runtime/picasso/src/ibc.rs (hunk 1 above)
Comment (kkast) [reply code/parachain/runtime/picasso/src/ibc.rs:155]: seems like it is correct, but if with or without testfast feature has the the same AdminOrigin and FreezeOrigin, maybe we shouldnt put them separately

**Review by kkast — APPROVED** [review]

**Review by kkast — APPROVED** [review]

**Pull request merged**